packages/web/src/components/collection/CollectionCard.tsx

```diff
@@ -51,50 +59,100 @@ export const CollectionCard = (props: CollectionCardProps) => {
     permalink,
     playlist_owner_id,
     repost_count,
-    save_count
+    save_count,
+    is_private,
+    access,
+    stream_conditions
   } = collection
 
+  const isPurchase = isContentUSDCPurchaseGated(stream_conditions)
+
+  const dogEarType = is_private
+    ? DogEarType.HIDDEN
+    : isPurchase && !access.stream
+    ? DogEarType.USDC_PURCHASE
+    : null
+
   return (
     <Paper
       role='button'
       tabIndex={0}
       onClick={handleClick}
-      aria-labelledby={`${id}-title ${id}-artist`}
+      aria-labelledby={`${id}-title ${id}-artist ${id}-repost ${id}-favorite ${id}-condition`}
       direction='column'
+      border='default'
       w={cardSizes[size]}
-      css={{ cursor: 'pointer' }}
+      css={{ cursor: 'pointer', overflow: 'unset' }}
     >
+      {dogEarType ? (
+        <DogEar type={dogEarType} css={{ top: -1, left: -1 }} />
+      ) : null}
       <Flex direction='column' p='s' gap='s'>
         <CollectionImage
           collectionId={id}
           size={cardSizeToCoverArtSizeMap[size]}
           data-testid={`${id}-cover-art`}
         />
-        <Link id={`${id}-title`} to={permalink} css={{ alignSelf: 'center' }}>
-          <Text variant='title' tag='span' color='default' textAlign='center'>
+        <Text
+          id={`${id}-title`}
+          variant='title'
+          color='default'
+          ellipses
+          asChild
+        >
+          <Link to={permalink} css={{ pointerEvents: 'none' }}>
             {playlist_name}
-          </Text>
-        </Link>
+          </Link>
+        </Text>
         <UserLink
           id={`${id}-artist`}
           userId={playlist_owner_id}
-          css={{ alignSelf: 'center' }}
+          textVariant='body'
+          css={{ justifyContent: 'center' }}
         />
       </Flex>
       <Divider orientation='horizontal' />
-      <Flex gap='l' p='s' justifyContent='center'>
-        <Flex gap='xs' alignItems='center'>
-          <IconRepost size='s' color='subdued' title={messages.repost} />
-          <Text color='subdued' variant='body' size='s'>
-            {repost_count}
-          </Text>
-        </Flex>
-        <Flex gap='xs' alignItems='center'>
-          <IconHeart size='s' color='subdued' title={messages.favorites} />{' '}
-          <Text color='subdued' variant='body' size='s'>
-            {save_count}
+      <Flex
+        gap='l'
+        p='s'
+        justifyContent='center'
+        backgroundColor='surface1'
+        borderBottomLeftRadius='m'
+        borderBottomRightRadius='m'
+      >
+        {is_private ? (
+          <Text
+            variant='body'
+            size='s'
+            strength='strong'
+            color='subdued'
+            id={`${id}-condition`}
+          >
+            {messages.hidden}
           </Text>
-        </Flex>
+        ) : (
+          <>
+            <Flex gap='xs' alignItems='center' id={`${id}-repost`}>
```

Comment:
nit: Very optional but I think you could make the aria text a slight bit better by adding this to this element
```tsx
aria-label={`{repost_count} {messages.repost}`}
```
so that it reads out as `10 reposts` instead of `reposts 10`

Reply:
(same for favorites)

Reply:
yeah we had actually discussed this. `10 reposts` reads like english, but `reposts 10` contextualizes the number better by the time you hear it. Unsure which is canonical for accessibility

Reply:
yeah im down to investigate more on this. personally i also like that we can use the actual elements labels rather than have to add a custom label, in the aria-label case.